app/models/ruby_method.rb

```diff
@@ -23,12 +23,24 @@ def object_constant
     body[:object_constant]
   end
 
-  def instance_method?
-    method_type == "instance_method"
+  TYPES = {
+    "class_method" => {
+      identifier: "::"
+    }.freeze,
+    "instance_method" => {
+      identifier: "#"
+    }.freeze
+  }.freeze
+
+  TYPES.each_key do |type|
```

Comment:
I'm not in favour of the meta-programming here.  If we were going to add new `TYPES` in the future I would see the benefit and be in favour of it, but we will only ever have 2 types. I would prefer to just define these methods plain-and-simple and use constants were applicable.

Reply:
> I'm not in favour of the meta-programming here. If we were going to add new `TYPES` in the future I would see the benefit and be in favour of it, but we will only ever have 2 types. I would prefer to just define these methods plain-and-simple and use constants were applicable.

I don't like code repeating, but OK, done.